beacon_chain/networking/eth2_network.nim

```diff
@@ -303,6 +308,16 @@ declareGauge nbc_gossipsub_good_fanout,
 declareGauge nbc_gossipsub_healthy_fanout,
   "numbers of topics with dHigh fanout"
 
+declareGauge nbc_torgossipsub_low_fanout,
+  "numbers of topics over tor with low fanout"
+
+declareGauge nbc_torgossipsub_good_fanout,
+  "numbers of topics over tor with good fanout"
+
+declareGauge nbc_torgossipsub_healthy_fanout,
+  "numbers of topics over tor with dHigh fanout"
+
+
```

Comment:
Extra newline? Doesn't really matter, maybe intentional (group `declareGauge`s, etc), but wasn't there before in that capacity.

Trivial nit, sure.